include/picongpu/param/transitionRadiation.param

```diff
@@ -0,0 +1,175 @@
+
+} // namespace linearFrequencies
+
+namespace logFrequencies
+{
+    namespace SI
+    {
+        //! mimimum frequency of the logarithmic frequency scale in units of [1/s]
+        constexpr float_64 omegaMin = 1.0e13;
+        //! maximum frequency of the logarithmic frequency scale in units of [1/s]
+        constexpr float_64 omegaMax = 1.0e17;
+    }
+
+    //! number of frequency values to compute in the logarithmic frequency [unitless] 
+    constexpr unsigned int nOmega = 256;
+
+} // namespace logFrequencies
+
+
+namespace listFrequencies
+{
+    //! path to text file with frequencies
+    constexpr char listLocation[] = "/path/to/frequency_list";
+    //! number of frequency values to compute if frequencies are given in a file [unitless]
+    constexpr unsigned int nOmega = 512;
+
+} // namespace listFrequencies
+
+
+    /** selected mode of frequency scaling:
+     *
+     * options:
+     * - linearFrequencies
+     * - logFrequencies
+     * - listFrequencies
+     */
+    namespace frequencies = logFrequencies;
+
+    ///////////////////////////////////////////////////
+
+
+    /** correct treatment of coherent radiation from macro particles
+     *
+     * Choose different form factors in order to consider different particle shapes for radiation
+     *  - macroParticleFormFactorCIC_3D ... CIC charge distribution
+     *  - macroParticleFormFactorTSC_3D ... TSC charge distribution
+     *  - macroParticleFormFactorPCS_3D ... PCS charge distribution
+     *  - macroParticleFormFactorCIC_1Dy ... only CIC charge distribution in y
+     *  - macroParticleFormFactorGauss_spherical ... symmetric Gauss charge distribution
+     *  - macroParticleFormFactorGauss_cell ... Gauss charge distribution according to cell size
+     *  - macroParticleFormFactorincoherent ... only incoherent radiation
+     *  - macroParticleFormFactorcoherent ... only coherent radiation
+     */
+    namespace macroParticleFormFactorCIC_3D { }
+    namespace macroParticleFormFactorTSC_3D { }
+    namespace macroParticleFormFactorPCS_3D { }
+    namespace macroParticleFormFactorCIC_1Dy { }
+    namespace macroParticleFormFactorGaussSpherical { }
+    namespace macroParticleFormFactorGaussCell { }
+    namespace macroParticleFormFactorIncoherent { }
+    namespace macroParticleFormFactorCoherent { }
+
+    namespace macroParticleFormFactor = macroParticleFormFactorGaussSpherical;
+
+    ///////////////////////////////////////////////////////////
+
+    namespace parameters
+    {
+        // number of observation directions 
+        constexpr unsigned int nPhi = 256;
+        constexpr unsigned int nTheta = 256;
+        constexpr unsigned int nObserver = nPhi * nTheta;
+
+        // theta goes from 0 to pi
+        constexpr float_64 thetaMin = 0.0; 
+        constexpr float_64 thetaMax = picongpu::PI; 
+
+        // phi goes from 0 to 2*pi
+        constexpr float_64 phiMin = 0.0;
+        constexpr float_64 phiMax = 2 * picongpu::PI;
```

Comment:
Your definition of the observation angle ranges is fine here, but that makes the other param file a bit obsolete. 
How about moving these definitions entirely to the observer param file?

Reply:
I would probably prefer using just a single param file, since `transitionRadiationObserver.param` is just used for creating unit-vectors in the PIC coordinatesystem from the `transitionRadiation.param` observation angles. 2 param files seems like a lot of files for this plugin with basically just 3x3 parameters (min, max, n) x (phi, theta, omega). But if the vector-creation moves into `transitionRadiation.param`, this file looks a little bit messy.

Reply:
I can understand both of your points. Please decide for yourself whether you prefer having two or just one param file. As long as the variable distribution is consistent (in contrast to the radiation plugin 😉 ) I am fine with it.